penpost/penpost-web
# Kb/feature/home component

#### Purpose:
 > Check all that apply.

### What does this PR do?
 > Clearly and concisely describe changes made. 
Added a home component so when a user goes to the website, they are taken to this page. I also changed App component into an ES6 arrow function.


Issues closed #

### Problems Encountered & Resulting Solutions  
> State any problems encountered 



> Describe your code changes in detail for reviewers.
Added a Home functional Component to display the welcome message, about us and buttons.


### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?
Home.js and Home.scss


### How should this be tested?
> What steps need to be taken to confirm functionality? 
npm start in the browser


### Steps To Take
> Any questions and possible steps moving forward.
